frontend/src/index.js

```diff
@@ -16,8 +16,22 @@ if (SENTRY_FRONTEND_DSN) {
   Sentry.init({
     dsn: SENTRY_FRONTEND_DSN,
     environment: ENVIRONMENT,
-    integrations: [new BrowserTracing()],
+    integrations: [
+      new BrowserTracing(),
```

Comment:
On a separate note (unrelated to the changes made in this PR), the `BrowserTracing` import from `@sentry/tracing` is now deprecated. It's recommended to directly import it from our framework-specific SDK, `@sentry/react`. This lets us remove the `@sentry/tracing` package altogether.